code/modules/mob/living/carbon/human/say.dm

```diff
@@ -5,22 +5,22 @@
 		verb_ask = pick(T.ask_mod)
 		verb_yell = pick(T.yell_mod)
 		verb_exclaim = pick(T.exclaim_mod)
+	if(wear_mask)
+		var/obj/item/clothing/mask/M = wear_mask
+		if(!isnull(M.chosen_tongue))
```

Comment:
Use ispath since this will be a path and not an object

Reply:
Are you sure you commented this on the right line?